lib/postgrex/connection.ex

```diff
@@ -140,33 +172,45 @@ defmodule Postgrex.Connection do
 
   ## Options
 
-    * `:timeout` - Call timeout (default: `#{@timeout}`)
+    * `:pool_timeout` - Time to wait in the queue for the connection
+    (default: `#{@pool_timeout}`)
+    * `:queue` - Whether to wait for connection in a queue (default: `true`);
+    * `:timeout` - Execute request timeout (default: `#{@timeout}`);
     * `:decode`  - Decode method: `:auto` decodes the result and `:manual` does
     not (default: `:auto`)
+    * `:decode_mapper` - Fun to map each row in the result to a term, see
```

Comment:
We dont need manual anymore. If we can make this backwards incompatible
change may be able to tidy up encode/decode a bit.

If we were to need it, likely it would make sense to add to db_connection.

On Monday, 4 January 2016, José Valim <notifications@github.com> wrote:

> In lib/postgrex/connection.ex
> <https://github.com/ericmj/postgrex/pull/128#discussion_r48723821>:
>
> >      * `:decode`  - Decode method: `:auto` decodes the result and `:manual` does
> >      not (default: `:auto`)
> > +    * `:decode_mapper` - Fun to map each row in the result to a term, see
>
> Do we care about making this public? Would using a mapper offer any
> advantage over decode: :manual? If so, maybe we should get rid of decode:
> :manual altogether?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ericmj/postgrex/pull/128/files#r48723821>.
>